README.md

```diff
@@ -280,10 +280,10 @@ which contains many examples you can follow with. Some key resources include:
 | **[Contribute]**    | Want to contribute? These instructions will help you get set up!                         |
 | **[Slack]**         | Have questions or issues unresolved? Join our community and ask away!                    |
 
-[Docs]: https://docs.lineapy.org/en/latest/index.html
-[Concepts]: https://docs.lineapy.org/en/latest/fundamentals/concepts.html
+[Docs]: https://docs.lineapy.org/en/lin-716-reorg-rtd-contents/index.html

```

Comment:
change back to main/index before merging.. applies to all.

Reply:
there are many places. search for lin-716 before merge.

Reply:
Done